saleor/checkout/tasks.py

```diff
@@ -109,3 +116,57 @@ def delete_expired_checkouts(
         else:
             task_logger.warning("Invocation limit reached, aborting task")
     return total_deleted, has_more
+
+
+@app.task
```

Comment:
IMHO, this should be on a separate celery queue (probably this one: `settings.py::CHECKOUT_WEBHOOK_EVENTS_CELERY_QUEUE_NAME`). In case having a lot of different events on queue, this could cause a delay in conversion, which could lead to unexpected behaviors (like missing orders, out of stock etc).

Reply:
Edit:  

> (probably this one: settings.py::CHECKOUT_WEBHOOK_EVENTS_CELERY_QUEUE_NAME)

Just saw `WEBHOOK` word  in the name, so probably we need different one